From 4dbbd93be5e9eedfcb8f32a4796f4b8f2d79403b Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 21 Nov 2013 10:10:28 +0800
Subject: [PATCH 265/602] ENGR00283354 init: enlarge maxinum number that can
 be locked

Enlarge maxinum number that can be locked in one process.
This commit is referring to ASUS&LGE's corresponding change.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx5x/init.rc    | 4 ++++
 imx6/etc/init.rc | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/imx5x/init.rc b/imx5x/init.rc
index 0dcd295..a74c47a 100644
--- a/imx5x/init.rc
+++ b/imx5x/init.rc
@@ -281,6 +281,10 @@ on post-fs-data
     # Set indication (checked by vold) that we have finished this action
     #setprop vold.post_fs_data_done 1
 
+on early-boot
+    # set RLIMIT_MEMLOCK to 64MB
+    setrlimit 8 67108864 67108864
+
 on boot
 # basic network init
     ifup lo
diff --git a/imx6/etc/init.rc b/imx6/etc/init.rc
index f048e14..654836b 100755
--- a/imx6/etc/init.rc
+++ b/imx6/etc/init.rc
@@ -294,6 +294,10 @@ on post-fs-data
     # Set indication (checked by vold) that we have finished this action
     setprop vold.post_fs_data_done 1
 
+on early-boot
+    # set RLIMIT_MEMLOCK to 64MB
+    setrlimit 8 67108864 67108864
+
 on boot
 # basic network init
     ifup lo
-- 
1.8.0

